ci-cd/Setup.md

```diff
@@ -7,15 +6,34 @@
 - Jenkins
 - TOMCAT
 
-### Machine Setup Instructions:
+# Machine Setup Instructions:
+
+### OS: Windows 7+
+### JDK Setup
+- [JDK 8 Download](http://www.oracle.com/technetwork/pt/java/javase/downloads/jdk8-downloads-2133151.html?printOnly=1)
+    - 64 bit OS : jdk-8u171-windows-x64.exe
+    - 32 bit OS : jdk-8u171-windows-i586.exe
+
+### Maven Setup
+- [Maven Download](http://apache.mirrors.pair.com/maven/binaries/)
+    - apache-maven-3.2.2-bin.zip 
```

Comment:
Why specifying an older version. Either don't specify or say x or newer version

Reply:
And if you are using for your own why not using latest?

Reply:
Agree. Changed this to apache-maven-3.5.3-bin.zip